diagnostic.md

```diff
@@ -23,7 +27,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('li').querySelectorAll('form');
```

Comment:
`.querySelectorAll()` is what you would use if you weren't using jQuery.  A few possible ways with jQuery are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.